pkg/trust/policy.go

```diff
@@ -131,8 +131,10 @@ func parseUids(colonDelimitKeys []byte) []string {
 				continue
 			}
 			parseduid := uid
-			if strings.Contains(uid, "<") && strings.Contains(uid, ">") {
-				parseduid = strings.SplitN(strings.SplitAfterN(uid, "<", 2)[1], ">", 2)[0]
+			ltidx := strings.Index(uid, "<")
```

Comment:
In this case I opted to implement the slicing here without `Cut`. Switching to `LastIndex` does slightly change behavior; arguably I could've changed it to do `strings.Index(parseduid[ltidx+1:], ">")` Maybe instead of `LastIndex` I should've used

Reply:
pkg/trust is security critical code, I am not familiar with this code but we should not chnage the behavior unless there is a proper reason.

Reply:
Updated code to maintain behavior